tests/UI/campaigns/functional/BO/14_advancedParameters/10_multistore/03_CRUDShops.ts

```diff
@@ -148,18 +148,8 @@ describe('BO - Advanced Parameters - Multistore : Create, Read, Update and Delet
     });
   });
 
-  // 4 : Delete shop URL
-  describe('delete shop URL', async () => {
-    it('should delete the shop URL', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'deleteShopURL', baseContext);
-
-      const textResult = await shopUrlPage.deleteShopURL(page, 1);
```

Comment:
Please fix eslint error => 'shopUrlPage' is defined but never used

Reply:
Fixed